backend/src/main/java/sullog/backend/alcohol/error/AlcoholControllerAdvice.java

```diff
@@ -0,0 +1,17 @@
+package sullog.backend.alcohol.error;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.bind.annotation.RestControllerAdvice;
+import sullog.backend.alcohol.error.exception.AlcoholException;
+
+@RestControllerAdvice
+public class AlcoholControllerAdvice {
+
+    @ExceptionHandler(AlcoholException.class)
+    ResponseEntity<?> exceptionHandler(AlcoholException e) {
```

Comment:
```suggestion
    ResponseEntity<ErrorCode> exceptionHandler(AlcoholException e) {
```

추후 개발하면서, 모든 케이스가 ErrorCode 타입으로 핸들링된다면 이렇게 수정되면 좋을 것 같습니다!

Reply:
넵 좋습니다!